__tests__/core/validation/validators/UrlValidator.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { UrlValidator } from '@/core/validation/validators'
+
+describe('UrlValidator.validate should', () => {
+  describe('return false when passing in', () => {
+    test('http://', () => {
+      expect(UrlValidator.validate('http')).toBeFalsy()
+    })
+    test('https://', () => {
+      expect(UrlValidator.validate('http://')).toBeFalsy()
+    })
+    test('https://foo', () => {
+      expect(UrlValidator.validate('https://q')).toBeFalsy()
+    })
+    test('https://foo..', () => {
+      expect(UrlValidator.validate('https://foo..')).toBeFalsy()
+    })
+    test('https://foo.-', () => {
+      expect(UrlValidator.validate('https://foo.-')).toBeFalsy()
+    })
+    test('https://foo--', () => {
+      expect(UrlValidator.validate('https://foo--')).toBeFalsy()
+    })
+    test('https://192.0.0', () => {
+      expect(UrlValidator.validate('https://foo--')).toBeFalsy()
```

Comment:
seems 'https://192.0.0' should be in validate()

Reply:
u r right! thanks